total_tolles_ferleihsystem/src/app/shared/rest/api.service.ts

```diff
@@ -119,18 +123,26 @@ export class ApiService implements OnInit {
 
     getRoot(): Observable<RootModel> {
         if (!this.rootSource.isStopped) {
-            let url = '/api'
-            if ((window as any).apiBasePath != null) {
-                url = (window as any).apiBasePath;
-            }
-            this.rest.get<RootModel>(url).subscribe(data => {
-                this.rootSource.next(data);
-                this.rootSource.complete();
+            this.webappSettingsService.getWebappSettings().subscribe((webappSettings: WebappSettings) => {
```

Comment:
either use take(1) operator in a pipe (before the subscribe) or unsubscribe observable when successfull to avoid memory leaks.